src/docfx/config/ops/OpsConfigAdapter.cs

```diff
@@ -114,6 +128,21 @@ public void Dispose()
             _http.Dispose();
         }
 
+        private string GetXrefMapApiEndpoint(string xrefEndpoint)
+        {
+            var isProduction = string.Equals(xrefEndpoint?.TrimEnd('/'), "https://xref.docs.microsoft.com", StringComparison.OrdinalIgnoreCase);
+            return isProduction
+                    ? "https://op-build-prod.azurewebsites.net"
+                    : "https://op-build-sandbox2.azurewebsites.net";
+        }
+
+        private async Task<string[]> GetXrefMaps(string xrefMapApiEndpoint, string tag, string xrefMapQueryParams)
+        {
+            var url = $"{xrefMapApiEndpoint}/v1/xrefmap{tag}{xrefMapQueryParams}";
+            var response = await Fetch(url, s_opsHeaders);
+            return JsonUtility.Deserialize<XrefMapApiResponse>(response, null).Links;
```

Comment:
nit: to save a class definition:

```suggestion
            return JsonConvert.DeserializeAnonymousType(
```

Reply:
how to describe the class with anonymous type, like `new { new[] { links = [] } }`, which is invalid

Reply:
```suggestion
            return JsonUtility.DeserializeAnonymousType(response, new { links = new [] { "" } });
```